database/migrations/2019_12_14_000001_create_personal_access_tokens_table.php

```diff
@@ -17,6 +17,7 @@ public function up(): void
             $table->string('name');
             $table->string('token', 64)->unique();
             $table->text('abilities')->nullable();
+            $table->text('data')->nullable();
```

Comment:
why not to create as ->json field here?

Reply:
@hsamshod just following the same as abilities column convention but yes it would be better to make it as json column